skiko/build-with-local-skia.sh

```diff
@@ -43,7 +43,7 @@ case $SKIA_TARGET in
     ;;
```

Comment:
As we mentioned in Slack, the additional Angle size is 14 mb. The Windows's Skiko x64 dll is 16.6 mb, so having Angle almost the same is suspicious.

Maybe we should find the right arguments for the compiler/linker to reduce the size? For example, for stripping unused code.

Reply:
> ANGLE depends on d3dcompiler_47.dll which is 8,4 MB alone

It looks like we should link with it, not include it?

Reply:
Does the binary includes only Angle or Skiko+Angle?

Reply:
> I've managed to teach angle-pack to build an "official build" (it's quite a challenge), but it actually became a bit bigger, most likely due to optimisations favouring speed over size.

I assume it is a static library `angle.lib`? If we link it to a dynamic library `skiko.dll`, or `angle.dll`, maybe there is an unused code, and we can strip it?

Reply:
> It looks like we should link with it, not include it?

Searched my PC, each app ships it separately :(.

P.S. x64 size is 4.7mb, arm64 size is 8.4

Reply:
> It looks like we should link with it, not include it?

No, it's a redistributable DLL from Microsoft, pretty much like vcruntime140.dll and stuff.

> Does the binary includes only Angle or Skiko+Angle?
> I assume it is a static library angle.lib? If we link it to a dynamic library skiko.dll, or angle.dll, maybe there is an unused code, and we can strip it?

It's ANGLE only, and those are 2 dynamic libraries: `libEGL.dll` and `libGLESv2.dll`. libEGL is small but libGLESv2 contains all the backend code.

Reply:
> P.S. x64 size is 4.7mb, arm64 size is 8.4

Indeed, I work on an Arm64 machine, so the sizes I get are for Arm64. `d3dcompiler_47.dll` is much bigger on Arm64 because it's an [ARM64X binary](https://learn.microsoft.com/en-us/windows/arm/arm64x-pe) (containing both native Arm64 code and x64-emulation-compatible Arm64EC code).

Reply:
@igordmn You can find the pre-built artifacts here: https://github.com/Alovchin91/angle-pack/releases/tag/66bc9cfa00

Reply:
@igordmn By the way, I've noticed that at least `skiko-windows-arm64.dll` built with Clang-CL is _almost 7 MB smaller_ than the MSVC one, 10,5 MB vs. 17,2 MB for Skiko 0.8.22.1.